### PR TITLE
(PC-19429) feat(accessibility): Make CulturalSurvey answers accessible

### PR DESCRIPTION
**Pull request by aliraiki**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-19429

## Checklist

I have:

- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](1)

## Screenshots

**delete** *if no UI change*

| Platform         | Before | After |
| :--------------- | :----: | :---: |
| iOS              |        |       |
| Android          |        |       |
| Phone - Chrome   |        |       |
| Tablet - Chrome  |        |       |
| Desktop - Chrome |        |       |
| Desktop - Safari |        |       |

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/culturalSurvey/components/CulturalSurveyCheckbox.tsx [hunk 1]
```diff
@@ -42,9 +36,18 @@ export const CulturalSurveyCheckbox = (props: CulturalSurveyCheckboxProps) => {
...
+    ? [theme.colors.secondary, theme.colors.primary]
+    : [theme.colors.greyMedium, theme.colors.greyMedium]
+
+  const accessibilityLabel = props.subtitle ? `${props.title} ${props.subtitle}` : props.title
   return (
-    <StyledLinearGradient>
-      <AnswerContainer onPress={onPress} testID={`${props?.title}-CulturalSurveyAnswer`}>
+    <StyledLinearGradient colors={colors}>
+      <AnswerContainer
+        accessibilityRole={AccessibilityRole.CHECKBOX}
```
Comment (pierrecanthelou) [inline src/features/culturalSurvey/components/CulturalSurveyCheckbox.tsx:47]: Est-e qu'on pourrait pas envisager des fonctions qui rempliraient les props pour certains types communs
`...accessibilityPropsCheckbox(accessibilityLabel, selected)`
=>
```
accessibilityRole={AccessibilityRole.CHECKBOX}
accessibilityState={{ checked: selected }}
accessibilityLabel={accessibilityLabel}
```

**Review by pierrecanthelou — APPROVED** [review]
Pour le reste rien à dire

File: src/features/culturalSurvey/components/CulturalSurveyCheckbox.tsx (hunk 1 above)
Comment (aliraiki) [reply src/features/culturalSurvey/components/CulturalSurveyCheckbox.tsx:47]: Très bonne idée effectivement

Comment (pierrecanthelou) [reply src/features/culturalSurvey/components/CulturalSurveyCheckbox.tsx:47]: Ok alors comment, dans votre process, on poursuis cette idée ?

Comment (aliraiki) [reply src/features/culturalSurvey/components/CulturalSurveyCheckbox.tsx:47]: Un ticket d'amélioration tech dans [ce board](https://passculture.atlassian.net/jira/software/c/projects/PC/boards/80)

**Pull request merged**